### PR TITLE
Update/term description

### LINKED ISSUE #237: UI should help people to learn core concepts
## Story

As a regular user who just started learning about NEAR, I want to know what I am looking at so I can build my understanding of the system.

Currently, most of the numbers in our UI are quite unintuitive (including NEAR, Gas, number of nodes, block height).

There already have been a few questions about what Tgas is (that is Teragas, which is 10^12 gas).

I believe we should have hints with short explanations and links to the documentation.

## Acceptance Criteria

- [x] Each term in the Detail page tables has a tooltip/modal with explanation
  - [x] Account Detail
  - [x] Transaction Detail
  - [x] Block Detail
  - [x] Dashboard
- [x] Mobile ok
- [ ] Tested

## Notes

List of terms and explanations: https://docs.google.com/spreadsheets/d/1-4-zKSvf_v68AGFFmTgoKoGxtRXLW5dL7L7wMEdAdZg/edit#gid=0

### PR DESCRIPTION
**Pull request by icerove**
fix #237

### CODE REVIEW

**Comment by icerove** [comment]
I redesign as Frol suggested.

File: frontend/src/components/accounts/AccountDetails.tsx [hunk 1]
```diff
@@ -25,12 +25,20 @@ export default class extends React.Component<Props> {
               title="Ⓝ Balance"
               text={<Balance amount={account.amount} />}
               className="border-0"
+              href={"https://docs.nearprotocol.com/docs/concepts/account"}
```
Comment (frol) [inline frontend/src/components/accounts/AccountDetails.tsx:28]: 1. Looking at this interface I would expect that we define the whole CardCell to become a link when I pass `href`, but it is just a link to the docs in the popup... I think it does not make a lot of sense to make this separation and it makes things easier to customize if you just explicitly include `<a href...>docs</a>` in the `termDescription`.

2. `href` and `termDesription` do not belong to `CardCell`. Let's not abuse it, unless it is absolutely necessary. To me, it makes a lot of sense to just have a standalone component `TermHelpButton`, and you just use it in the title:

    ```jsx
            <CardCell
              title={
                <>
                  {"Ⓝ Balance "}
                  <TermHelpButton>
                    {"Total NEAR token balance. This includes the 'Locked' balance. "}
                    <a href="https://docs.nearprotocol.com/docs/concepts/account">docs</a>
                  </TermHelpButton>
                </>
              }
              text={<Balance amount={account.amount} />}
              className="border-0"
            />
    ```

**Comment by frol** [comment]
@icerove The on-hover implementation looks great to me. Though, it still breaks some of the cells on Dashboard... I will just add a separate issue about that as part of #201 since we need some redesign.

**Comment by icerove** [comment]
@frol 
>Looking at this interface I would expect that we define the whole CardCell to become a link when I pass href, but it is just a link to the docs in the popup
the wallet design is like that. We have the agreement to follow the wallet design. FurtherMore, if the whole cell is the link, how should we show the Modal and description? when click the cell, it will trigger link and cover the button to show the description.

**Comment by icerove** [comment]
@frol 
>href and termDesription do not belong to CardCell. Let's not abuse it, unless it is absolutely necessary. 

Why? Every term description is in the `CardCell` and We do not have other to do so.

**Comment by frol** [comment]
> the wallet design is like that. We have the agreement to follow the wallet design. FurtherMore, if the whole cell is the link, how should we show the Modal and description? when click the cell, it will trigger link and cover the button to show the description.

There is a misunderstanding.

I implied that it does not make any sense to me to have `href` for the CardCell as it is confusing (and I specifically described why I find that confusing).

I don't want to have either `href` or `termDescription` on CardCell component.

I want to have a separate `HelpButton` component doing exactly the thing it is designed to do:

1. display the small (i) button
2. handle the onClick event presenting the help message

**Comment by frol** [comment]
> Why? Every term description is in the `CardCell` and We do not have other to do so.

Please, separate concerns as otherwise, we will end up with a monster component with millions of props for all sorts of cases.

File: frontend/src/components/accounts/AccountDetails.tsx [hunk 2]
```diff
@@ -22,20 +23,64 @@ export default class extends React.Component<Props> {
         <Row noGutters>
           <Col md="3">
             <CardCell
-              title="Ⓝ Balance"
+              title={
+                <>
+                  {"Ⓝ Balance"}
+                  <TermHelperButton title={"Balance"}>
```
Comment (frol) [inline frontend/src/components/accounts/AccountDetails.tsx:29]: I clearly did not see the duplication. Since it is my fault, I will fix this.

**Pull request merged**